docs/criteria/connectors/rc.md

```diff
@@ -0,0 +1,143 @@
+
+Some connectors are unable to support TPC derived test packages due to their nature (e.g. GitHub Issues).
+These connectors are exempt from running the TPC derived test packages, and rely instead on their general integration test.
+
+This table defines the required features and/or tests for each connector:
+
+| Connector     | [TPC-H Derived Tests (Scale Factor)](#tpc-h) | [TPC-DS Derived Tests (Scale Factor)](#tpc-ds) | [Federation](#federation) | [Data Correctness](#data-correctness) | [Streaming](#streaming) | [Native Schema Inference](#schema-inference) |
+| ------------- | - | - | - | - | - | - |
+| Clickhouse    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Databricks    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Delta Lake    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Dremio        | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| File          | ✅ (1) | ✅ (0.5) | ❌ | ❌ | ❌ | ❌ |
+| FTP/SFTP      | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| GraphQL       | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| GitHub        | ❌ | ❌ | ⚠️ | ❌ | ❌ | ❌ |
+| HTTP/HTTPS    | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| MS SQL        | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| MySQL         | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| ODBC          | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| PostgreSQL    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Sharepoint    | ❌ | ❌ | ⚠️ | ❌ | ❌ | ❌ |
+| Snowflake     | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Spice.AI      | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| S3            | ✅ (1) | ✅ (0.5) | ⚠️ | ❌ | ❌ | ❌ |
+| Azure BlobFS  | ✅ (1) | ✅ (0.5) | ⚠️ | ❌ | ❌ | ❌ |
+| Spark         | ✅ (1) | ✅ (0.5) | ✅ | ✅ | ✅ | ✅ |
+
+### All Connectors
+
+These requirements are imposed on every connector, regardless of the selected features/tests in the above table.
+
+- [ ] [Core Arrow Data Types](../definitions.md) are supported
+- [ ] For exceptions where a [Core Arrow Data Type](../definitions.md) is not supported, print a warning and ignore the column instead of crashing or failing the query.
+- [ ] All known [Major Bugs](../definitions.md) are resolved
+
+#### Documentation
+
+- [ ] Documentation includes all information and steps for a user to set up the connector.
+- [ ] Documentation includes all known issues/limitations for the connector.
+- [ ] Documentation includes any exceptions made to allow this connector to reach RC quality (e.g. if a particular data type cannot be supported by the connector).
+- [ ] The connector has an easy to follow quickstart.
+
+#### Testing
+
+- [ ] Integration tests to cover simple use cases based on the connector type, e.g. a Spicepod sourcing a file from an FTP server, reading latest GitHub issues, etc.
+
+### Conditional Criteria
+
+The following features/tests are dependent on the required features/tests for the specified connector, from the [Connector criteria table](#rc-release-criteria)
+
+#### Streaming
+
+- [ ] Data is streamed when reading/performing queries from this connector.
+
+#### Schema Inference
```

Comment:
I think an important alternate condition for this (for some connectors with a _"fixed"_ schema), is that schema inference is determined natively OR the schema requires no inference.